src/components/shopping-cart/container/index.js

```diff
@@ -1,18 +1,36 @@
+// @flow
+
+import type { DispatchType } from 'constants/redux'
+
 import * as ReactRedux from 'react-redux'
 import * as Reselect from 'reselect'
 
 import selectors from 'redux/selectors'
-import ShoppingCart from '../index'
+import shoppingCartAction from 'redux/actions/shopping-cart'
+
+import * as ShoppingCart from '../index'
+
+type OwnPropsType = {||}
```

Comment:
Is this for use later or is it filling a function now?

Reply:
For later use, and also referenced in
```js
type PropsType = {
  ...OwnPropsType,
  ...ShoppingCart.PropsType,
}
```
and redux connect

Reply:
this should really be `ReactRouter.ContextRouter` since the container is a link thougth the <Route> which sets these props